PayloadPkg/FirmwareUpdate/FirmwareUpdate.c

```diff
@@ -188,6 +189,24 @@ VerifyFwVersion (
     }
   }
 
+  //
+  // Check SVN for ACM update
+  //
+  if (((UINT32)ImageHdr->UpdateHardwareInstance) == FW_UPDATE_COMP_ACM0_REGION) {
+    DEBUG((DEBUG_INFO, "Capsule update is for ACM region!!\n"));
+    Status = CheckAcmSvn (ImageHdr, &SvnStatus);
```

Comment:
Why do we need SvnStatus, we can just use return type, return success or error

Reply:
Raghava, there would three cases here. SVN check pass, fail and error where it fails to compute SVN. This would make sure right error cases would be passed to caller.

Reply:
I guess it all just converts to EFI_STATUS anyways. Got rid of SvnStatus.